test/real_greater_than_operator_test.cpp

```diff
@@ -162,17 +162,17 @@ TEST_CASE("Operator >") {
 
         SECTION("With precision exception") {
             SECTION("Explicit > Algorithm") {
-                boost::real::real a("1.11");
+                boost::real::real a("1.111");
                 boost::real::real b(one_one_one, 1);
 
-                CHECK_THROWS_AS(a > b, boost::real::precision_exception);
```

Comment:
The idea of the test is to have a precision error, that is why "a = 1.11" that is equal to "b", but because "b" is an algorithmic number, then, we can't know whether "a > b" or not.

If you change a to 1.111 you know that a is greater than b because you know that 1.111.... > 1.110.... for all the possible values of the remaining infinity lower digits of "b".

Reply:
Yeah, same as before. I thought one_one_one is 1.111111... 
Will fix